_posts/en/newsletters/2019-10-16-newsletter.md

```diff
@@ -0,0 +1,223 @@
+      previously only used in [bip-taproot][] and [bip-tapscript][].
+
+    - The 10,000-byte and 201 non-push opcode limits have been dropped
+      from [bip-tapscript][] (mentioned in [#65][tapscript resource
+      limits]).
+
+    - The maximum depth of the merkle tree has been increased from 32
+      levels to 128 levels.
+
+    Wuille's email and the updated BIPs provide rationales for each of
+    these decisions as well as links to previous discussions about them.
+
+- **Proposed default LN fee increase:** Rusty Russell proposed an
+  increase in the default in-channel fees to be used by nodes, going
+  from 1,000 millisatoshis (msat) plus 1 part-per-million (ppm) to 5,000
+  msat plus 500 ppm.  He notes that the current defaults don't provide
+  much room for allowing users to lower fees and the popularity of nodes
+  currently charging higher fees indicates many users are currently not
+  sensitive to fees.  His email provides a chart estimating the old and
+  new costs of transferring various amounts of money in USD terms at
+  $10,000 USD/BTC:
+
+    | Amount |  Before     | After       |
+    |--------|-------------|-------------|
+    | 0.1c   |  0.0100001c |  0.05005c   |
+    | 1c     |  0.010001c  |  0.0505c    |
+    | 10c    |  0.01001c   |  0.055c     |
+    | $1     |  0.0101c    |  0.1c       |
+    | $10    |  0.011c     |  0.55c      |
+    | $100   |  0.02c      |  5.05c      |
+    | $1000  |  0.11c      |  50.05c     |
+
+    Fellow C-Lightning maintainer ZmnSCPxj and Eclair maintainer
+    Pierre-Marie Padiou indicated support for the proposal.  LND
+    maintainer Olaoluwa Osuntokun believed the reasoning behind the
+    proposal had several flaws and advocated instead to "educate
+    prospective routing node operators on best practices, provide
+    analysis tools [...], and leave it up to the market participants to
+    converge on steady state economically rational fees."  As of this
+    writing, discussion about the topic remains ongoing.
+
+- **Conference summary: Cryptoeconomic Systems Summit:** this conference
+  held last weekend on the MIT campus covered a variety of topics
+  about ensuring cryptocurrencies are both useful and secure.  Videos
+  are now available courtesy of [the DCI's YouTube channel][css vids]
+  and [transcripts][css ts] for several of the talks have been provided
+  by Bryan Bishop and others.  Out of the subset of talks that were
+  [transcribed][css ts], we thought the following three topics might be
+  of particular technical interest to the readers of this newsletter:
+
+    - *Everything is broken* by Cory Fields ([transcript][fields ts], [video][fields vid]), a
+      talk that describes how Bitcoin is at risk not just from its own
+      software bugs but also from the bugs introduced into the
+      libraries, operating systems, and even hardware that it depends
+      upon.  Fields then looks back in time when a profligation of
```

Comment:
While profligate is a great word, I don't think this use of profligation is quite correct. I had a hard time finding evidence of profligation used in this way elsewhere. Happy to be corrected.